src/api/tables.ts

```diff
@@ -151,8 +151,8 @@ const getTablesSql = (sqlTemplates) => {
       FROM
         relationships
       WHERE
-        (relationships.source_schema = tables.schema AND relationships.source_table_name = tables.name)
-        OR (relationships.target_table_schema = tables.schema AND relationships.target_table_name = tables.name)`
+        (relationships.source_schema :: text = tables.schema AND relationships.source_table_name :: text = tables.name)
+        OR (relationships.target_table_schema :: text = tables.schema AND relationships.target_table_name :: text = tables.name)`
```

Comment:
`source_schema` is `regnamespace`, need to convert to do comparison. Same for others.